plugins/SitesManager/Controller.php

```diff
@@ -335,4 +339,10 @@ private function getVueInitializeCode($vueVersion = '3')
 window._paq.push(['trackPageView']); // To track a page view
 INST;
     }
+
+    private function getReactCode()
+    {
+        $request = \Piwik\Request::fromRequest();
+        $siteId = $request->getIntegerParameter('idSite', 1);
+    }
```

Comment:
How does this produce the react code? If it doesn't and just returns the site ID, we probably should name it accordingly and add a comment why it's done this way compared to the other JS libraries.

Reply:
@michalkleiner I forgot to remove this unwanted code, removed